tests/_system/test_onedim_ckd_vs_mono.py

```diff
@@ -0,0 +1,375 @@
+        out=np.full_like(brf_mono_averaged.values, fill_value=np.nan),
+    )
+    rmad = np.mean(rad[~np.isnan(rad)])
+    minrad = np.min(rad[~np.isnan(rad)])
+    maxrad = np.max(rad[~np.isnan(rad)])
+    text = (
+        f"mean(RAD) >= {round(100 * rmad, 2)} %\n"
+        f"min(RAD) >= {round(100 * minrad, 2)} %\n"
+        f"max(RAD) >= {round(100 * maxrad, 2)} %"
+    )
+    plt.text(
+        0.5,
+        0.9,
+        text,
+        ha="center",
+        va="center",
+        transform=plt.gca().transAxes,
+    )
+
+    plt.legend(["mono_double (averaged)", "ckd_double"])
+    plt.title(f"Wavelength bin: {wavelength_bin}, reflectance = {reflectance}")
+    plt.tight_layout()
+    fig.savefig(fname_plot, dpi=200)
+    plt.close()
+
+
+@pytest.mark.parametrize("reflectance", [0.0, 0.5, 1.0])
+@pytest.mark.skipif_data_not_found("absorption_spectrum", "H2O-spectra-18000_18100")
+@pytest.mark.skipif_data_not_found("absorption_spectrum", "H2O-spectra-18100_18200")
+@pytest.mark.skipif_data_not_found("absorption_spectrum", "H2O-spectra-18200_18300")
+@pytest.mark.skipif_data_not_found("absorption_spectrum", "H2O-spectra-18300_18400")
+@pytest.mark.slow
+def test_550(reflectance):
+    r"""
+    Results consistency between `mono_double` and `ckd_double` modes
+    ================================================================
+
+    This test checks the results consistency between mono_double and ckd_double
+    modes in the [545, 555] nm wavelength bin.
+
+    Rationale
+    ---------
+
+    * Sensor: Distant measure covering a plane (11 angular points,
+      10000 sample per pixel) and targeting (0, 0, 0).
+    * Illumination: Directional illumination with a zenith angle
+      :math:`\theta = 50.0°`.
+    * Surface: a square surface with a Lambertian BRDF with reflectance
+      parameter in :math:`\rho \in [0.0, 0.5, 1.0]`.
+    * Atmosphere: a molecular atmosphere derived from the AFGL (1986)
+      thermophysical properties `us_standard` model, with both scattering and
+      absorption enabled.
+    * Integrator: volumetric path tracer.
+
+    Expected behaviour
+    ------------------
+
+    The [545, 555] nm - integrated BRF results obtained with the `mono_double`
+    and `ckd_double` modes should agree within 5%.
+
+    The `mono_double` BRF results are spectrally averaged according to the
+    following formula:
+
+    .. math::
+
+       \hat{q} = \frac{
+          \int_{
+             \lambda_{\mathrm{min}}
+          }^{
+             \lambda_{\mathrm{max}}
+          }
+          q(\lambda) \, \mathrm{d} \lambda
+       }{
+          \int_{
+             \lambda_{\mathrm{min}}
+          }^{
+             \lambda_{\mathrm{max}}
+          }
+          \mathrm{d} \lambda
+       }
+
+    where
+
+    * :math:`q` is some spectral quantity (e.g. the BRF)
+    * :math:`\lambda` denotes the wavelength
+    * :math:`[\lambda_{\mathrm{min}},\, \lambda_{\mathrm{max}}]` is the
+      wavelength bin
+
+    Results
+    -------
```

Comment:
I suggest also annotating the plots
```suggestion
    -------
    
    Note: :math:`\mathrm{RAD}(x, y) = \left| \dfrac{x - y}{x} \right|`

```